core/src/test/java/io/confluent/kafka/schemaregistry/ClusterTestHarness.java

```diff
@@ -147,7 +149,7 @@ public void setUp() throws Exception {
                                               getSecurityProtocol());
 
     if (setupRestApp) {
-      restApp = new RestApp(choosePort(), zkConnect, KAFKASTORE_TOPIC, compatibilityType);
+      restApp = new RestApp(choosePort(), zkConnect, KAFKASTORE_TOPIC, compatibilityType, groupId);
```

Comment:
mmm... this works but will require adding more and more arguments as we test more configurations.
I was thinking you can test the KafkaStore directly and use this util to just pass in a properties file that contains this configuation:
StoreUtils.createAndInitKafkaStoreInstance(String zkConnect, ZkClient zkClient, Store<String, String> inMemoryStore,Properties props)

Would this work?